src/theme.js

```diff
@@ -1,8 +1,9 @@
 // SIZE contstants
+const PAGE_WIDTH_PX = 1020
+
 export const REM_PX = 16
-export const PAGE_WIDTH_PX = 960
-export const GUTTER_PX = 30
-export const PAGE_WIDTH_REM = PAGE_WIDTH_PX / REM_PX
+export const GUTTER_PX = 10
+export const PAGE_WIDTH_REM = `${PAGE_WIDTH_PX / REM_PX}rem`
```

Comment:
Obviously I don't have a full understanding of things but, do you need the **rem** on the end? as you took it out earlier.

Reply:
We don't intend to use this value as number, so to help with that pattern we add it here so it's harder to do anything with that as a _number_